services/dy-jupyter/docker-compose.yml

```diff
@@ -27,25 +27,25 @@ services:
           "output_2":{"label": "output 2", "displayOrder":1, "description": "Output 2", "type": "data:*/*"}
           }}'
         simcore.service.settings: '[
-                                    {"name": "ports", "type": "int", "value": 8888},
-                                    {"name": "constraints", "type": "string", "value": ["node.platform.os == linux"]},
-                                    {"name": "resources", "type": "resources", "value":
-                                      {
-                                        "Limits": {
-                                          "NanoCPUs": 4000000000,
-                                          "MemoryBytes": 8589934592
-                                        }
-                                      }
-                                    }
-                                  ]'
+          {"name": "ports", "type": "int", "value": 8888},
+          {"name": "constraints", "type": "string", "value": ["node.platform.os == linux"]},
+          {"name": "resources", "type": "resources", "value":
```

Comment:
I have the impresssion some formatting went south here :-)

Reply:
ahah... well I don't take care about formatting since black does it  for me automatically ;)